index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.min.css" integrity="sha512-q3eWabyZPc1XTCmF+8/LuE1ozpg5xxn7iO89yfSOd5/oKvyqLngoNGsx8jq92Y8eXJ/IRxQbEC+FGSYxtk2oiw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Raymond's Portfolio</title>
+</head>
+<body>
+  <header>
+    <!-- Navigation Bar -->
+    <nav>
+      <a href="#" class="logo">
+        Ray
+        <img src="./images/r_logo.png" alt="logo" width="20px" height="20px">
+      </a>
+
+      <span class="menu-btn"><i class="fas fa-bars"></i></span>
+    </nav>
+
+    <!-- Headline Section -->
+    <section class="headline">
+      <h1>
+        I'm Mariane Glad to see you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.
+      </p>
```

Comment:
- [ ] Kindly update the text `color` and `font-size` of this paragraph to match the one in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/212065982-97b7b7bf-7ece-42ed-beaa-9dae533664be.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/212066010-f36335cb-8e5c-4b89-b6b9-0a9888553cd2.png)